extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/photos/ImgurPhotosImporter.java

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
delete this since we aren't dealing with import yet

Reply:
Done